src/pacing.py

```diff
@@ -55,15 +55,13 @@ class Binning(PacingFunction):
     def __init__(self, args):
         super(Binning, self).__init__()
         self.bin_variable = args.bin_variable
+        self.descending = args.descending  
         self.bin_method = args.bin_method
         self.n_bins = args.n_bins
         self.bin_validation_metric = args.bin_validation_metric
 
+    #def __call__(self, df, descending=False):
     def __call__(self, df):
-        df = df.sort_values(by=[self.bin_variable])
-
-        print("--------------------------")
```

Comment:
I think I'd rather have these two prints back, for better logging.

Reply:
Of course!